src/js/view/controls/rightclick.js

```diff
@@ -23,21 +23,24 @@ export default class RightClick {
     buildArray() {
         const semverParts = version.split('+');
         const majorMinorPatchPre = semverParts[0];
+        const model = this.model;
+        const poweredBy = model.get('localization').poweredBy;
+        const versionSpan = `<span class="jw-reset">JW Player ${majorMinorPatchPre}</span>`;
 
         const menu = {
             items: [{
                 type: 'info'
             },
             {
-                title: `${this.model.get('localization').poweredBy} <span class="jw-reset">JW Player ${majorMinorPatchPre}</span>`,
+                title: model.get('languageDir') === 'rtl' ? `${versionSpan} ${poweredBy}` : `${poweredBy} ${versionSpan}`,
```

Comment:
I don't think it makes sense to add a config property for this one if statement. Where else and what else is it used for?

Why don't you just call `isRtl(config.language)` here.